docs/index.adoc

```diff
@@ -1,16 +1,24 @@
+:experimental:
+include::_assets/attributes.adoc[]
+
+:context: proxy
+
+[id="using-book-{context}"]
 = Kroxylicious Proxy
-3.0, July 29, 2022: AsciiDoc article template
-:toc:
-:icons: font
-:url-quickref: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/
-:source-highlighter: pygments
 
-include::overview.adoc[leveloffset=1]
+include::assemblies/assembly-overview.adoc[leveloffset=+1]
 
-include::available-filters.adoc[leveloffset=1]
+//built-in filters
```

Comment:
I think the filters ought to be grouped together in their own section 'filters'.  It seems weird for 'record encryption' to be at the same level as 'operating proxies'